templates/deployments.yaml

```diff
@@ -409,13 +409,27 @@ spec:
         - configMapRef:
             name: {{ .Release.Name }}-st2-urls
             optional: true
+    {{- if .Values.st2web.config }}
+        volumeMounts:
+          - name: st2web-config-vol
+            mountPath: /opt/stackstorm/static/webui/config.js
+            subPath: st2web.config.js
+    {{- else }}
         volumeMounts: []
+    {{- end }}
         resources:
 {{ toYaml .Values.st2web.resources | indent 10 }}
     {{- if .Values.st2web.serviceAccount.attach }}
       serviceAccountName: {{ template "stackstorm-ha.serviceAccountName" . }}
     {{- end }}
+    {{- if .Values.st2web.config }}
+      volumes:
+        - name: st2web-config-vol
+          configMap:
+            name: {{ .Release.Name }}-st2web-config
```

Comment:
To make sure the st2web deployment redeploys the pods on config change, we'll need to include the Configmap checksum via  `annotations`.

Take a look at other examples like 
https://github.com/StackStorm/stackstorm-ha/blob/1c5103880f2e80f9ae5eb1d97c218aa7717a3258/templates/deployments.yaml#L33-L35

Reply:
done :)